backend/src/main/java/team/teamby/teambyteam/notice/presentation/NoticeController.java

```diff
@@ -0,0 +1,36 @@
+package team.teamby.teambyteam.notice.presentation;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import team.teamby.teambyteam.member.configuration.AuthPrincipal;
+import team.teamby.teambyteam.member.configuration.dto.MemberEmailDto;
+import team.teamby.teambyteam.notice.application.NoticeService;
+import team.teamby.teambyteam.notice.application.dto.NoticeRegisterRequest;
+
+import java.net.URI;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/team-place")
+public class NoticeController {
+
+    private final NoticeService noticeService;
+
+    @PostMapping("/{teamPlaceId}/feed/notice")
+    public ResponseEntity<Void> register(
+            @RequestBody @Valid final NoticeRegisterRequest reqeust,
+            @PathVariable final Long teamPlaceId,
+            @AuthPrincipal final MemberEmailDto memberEmailDto) {
+
```

Comment:
필립은 ) {도 개행하는 것 같아요
```suggestion
    public ResponseEntity<Void> register(
            @RequestBody @Valid final NoticeRegisterRequest reqeust,
            @PathVariable final Long teamPlaceId,
            @AuthPrincipal final MemberEmailDto memberEmailDto
    ) {

```
근데 이게 안되어 있는 부분이 다른데도 많네요(저포함)